mllib/src/main/scala/org/apache/spark/mllib/clustering/KMeans.scala

```diff
@@ -558,6 +475,7 @@ object KMeans {
    * Trains a k-means model using specified parameters and the default values for unspecified.
    */
   @Since("0.8.0")
+  @deprecated("Use train method without 'runs'", "2.1.0")
```

Comment:
I think we should add them for completeness, and deprecate all overloads using `runs`.